example_notebooks/6_mls_wheat_demo.ipynb

```diff
@@ -0,0 +1,2666 @@
+{
```

Comment:
<p>I don't think further details are necessary, because there is nothing too specific for this survey.. We could add links to the Wiki throughout the notebook(s) though :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/3dgeo-heidelberg/helios/pull/303/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>